lib/ui/views/edit_dish/edit_dish_viewmodel.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:developer';
+
+import 'package:receipe_app/ui/views/edit_dish/edit_dish_view.form.dart';
+import 'package:stacked/stacked.dart';
+
+class EditDishViewModel extends FormViewModel {
+  List<String> _allIngredients = [];
+  List<String> get allIngredients => _allIngredients;
+
+  void addTextFieldInputToList(String? input) {
+    log('$ingredientsValue ');
```

Comment:
log is from dart:developer ??

Reply:
yeah, forgot to remove that

Reply:
please do, make use of the logger instance we have: `final _logger = getLogger("your-class-name");` and use with _logger.i("what you want logged");

Reply:
ii have removed. it